docs/esp8266/general.rst

```diff
@@ -183,13 +184,16 @@ limitation with usage of TLS on the low-memory devices:
    accessing various REST APIs, which usually require much smaller messages.
    The buffers size is on the order of 5KB, and is adjusted from time to
    time, taking as a reference being able to access https://google.com .
-   The smaller buffer hower means that some sites can't be accessed using
-   it, and it's not possible to stream large amounts of data.
+   The smaller buffer however means that some sites can't be accessed using
```

Comment:
Yeah, I started to write about max fragment size extension, except I didn't make such bold claims as "no HTTPS website does" (rather "current implementations of popular server TLS libraries doesn't support it"). But then I thought: why adding more length to already long section? It's targeted at the end users, if it's not supported, why burden them with technical details?